### PR TITLE
Fr us 17 refactor

### LINKED ISSUE #123: US 17 - Disabled items still showing up on items index page

### PR DESCRIPTION
**Pull request by fredrondina**
Changed items index method so that when /items is being accessed it only displays active items.

updated tests to reflect these changes

closes #123

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def index
       @merchant = Merchant.find(params[:merchant_id])
       @items = @merchant.items
     else
-      @items = Item.all
+      @items = Item.where(active?: true)
```
Comment (kelshadarby) [inline app/controllers/items_controller.rb:8]: Nice catch!

**Review by kelshadarby — APPROVED** [review]

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (brian-greeson) [inline app/controllers/items_controller.rb:8]: Great catch but I think this should live in the model?

File: spec/features/items/index_spec.rb [hunk 2]
```diff
@@ -19,8 +19,7 @@
       expect(page).to have_link(@tire.merchant.name)
       expect(page).to have_link(@pull_toy.name)
       expect(page).to have_link(@pull_toy.merchant.name)
-      expect(page).to have_link(@dog_bone.name)
-      expect(page).to have_link(@dog_bone.merchant.name)
+      expect(page).to_not have_link(@dog_bone.name)
```
Comment (brian-greeson) [inline spec/features/items/index_spec.rb:22]: Great catch

**Review by brian-greeson — APPROVED** [review]
I like it. We can move the AR call into the model on the next refactor

**Pull request merged**